README.md

```diff
@@ -38,12 +38,26 @@ pip install openvino-tokenizers[transformers]
 conda install -c conda-forge openvino openvino-tokenizers && pip install transformers[sentencepiece] tiktoken
 ```
 
-### Build and install from source after [OpenVINO installation](https://docs.openvino.ai/2023.2/openvino_docs_install_guides_overview.html)
+### Install Pre-Release Version
+
+Use `openvino-tokenizrs[transformers]` to install tokenizers conversion dependencies.
+```bash
+pip install --pre -U openvino openvino-tokenizers --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

Comment:
openvino will be installed as dependency, no needs to set it manually for installation

Reply:
Yes, but this way it will be updated to the same dev version as tokenizers.